packages/connectivity/src/scp-cf/jwt/jwt.ts

```diff
@@ -62,18 +68,39 @@ export function getTenantId(
   return decodedJwt.zid || decodedJwt.app_tid || undefined;
 }
 
+/**
+ * Check if the given JWT is an IAS token.
+ * Currently, there are only two domains for IAS tokens:
+ * `accounts.ondemand.com` and `accounts400.ondemand.com`.
+ * @param decodedJwt - The decoded JWT to check.
+ * @returns Whether the given JWT is an IAS token.
+ * @internal
+ */
+export function isIasToken(decodedJwt: JwtPayload): boolean {
+  if (!decodedJwt.iss) {
+    return false;
+  }
+  try {
+    const issUrl = new URL(decodedJwt.iss);
+    const hostname = issUrl.hostname.toLowerCase();
+    return (
+      hostname.endsWith('.accounts.ondemand.com') ||
+      hostname.endsWith('.accounts400.ondemand.com')
+    );
+  } catch {
+    return false;
+  }
+}
+
 /**
  * Check if the given JWT is not an IAS token.
  * Currently, there are only two domains for IAS tokens:
- * `accounts.ondemand.com` and `accounts400.onemand.com`.
+ * `accounts.ondemand.com` and `accounts400.ondemand.com`.
  * @param decodedJwt - The decoded JWT to check.
  * @returns Whether the given JWT is not an IAS token.
  */
 function isNotIasToken(decodedJwt: JwtPayload): boolean {
```

Comment:
[req] Remove this function and replace it's usages with `!isIasToken()` instead.